assignments/result-option-assignment.adoc

```diff
@@ -21,7 +21,7 @@ The variants of the Result type are `Ok()` and `Err(e)`. It is used to handle er
 If an operation was successful, `Ok()` is returned. `Ok()` can be empty or have a
 return value. `Err(e)` contains an error message that can be printed.
 
-# `match`
+=== `match`
```

Comment:
I think this’ll give h3 nested into h1 in html and that iirc strictly speaking is wrong. If this is just xtra info, perhaps admonition block would work best: https://asciidoctor.org/docs/user-manual/#admonition ?

The # headers are leftover from markdown days, we indeed should use = now

Reply:
Oooh, nice! Done; result looks like this:
<img width="1122" alt="Bildschirmfoto 2020-05-15 um 15 46 22" src="https://user-images.githubusercontent.com/581552/82057277-62499500-96c3-11ea-96a4-edf5a65a2a64.png">